posts/improving-the-access-control-of-a-jhipster-application.md

```diff
@@ -0,0 +1,222 @@
+---
+authors:
+- Theo Lebrun
+tags:
+- JHipster
+- Spring
+- Spring Security
+- React
+- Angular
+date: 2019-01-22T15:24:45.000Z
+title: "Improving the access-control of a JHipster application"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2019/01/access-control.png
+---
+
+By default, a JHipster application will give any authenticated user the right to perform the four basic functions (or CRUD) on every entities. This behavior is very useful to showcase the application since any user can for example create an entity and update it. However, for a real application in production that kind of behavior is not really appropriate. The application's routes and API need to be restricted so a regular user does not have total freedom on every entities.
```

Comment:
```suggestion
By default, a JHipster application will give any authenticated user the right to perform the four basic functions (or CRUD) on every entity. This behavior is very useful to showcase the application since any user can for example create an entity and update it. However, for a real application in production that kind of behavior is not really appropriate. The application's routes and API need to be restricted so a regular user does not have total freedom on every entity.
```